modules/viewing-pipeline-run-step-logs.adoc

```diff
@@ -0,0 +1,49 @@
+:_module-type: PROCEDURE
+
+[id="viewing-pipeline-run-step-logs_{context}"]
+= Viewing pipeline run step logs
+
+[role='_abstract']
+To help you troubleshoot and audit your pipelines, you can review and analyze the log of each pipeline step using the log viewer. From here, you can search for specific log messages, download the logs for each step in your pipeline. If the pipeline is running, you can also pause and resume the log from the log viewer.   
+
+.Prerequisites
+* You have installed the OpenShift Pipelines operator.
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* You have previously created a data science project that is available and contains a pipeline server.
+* You have imported a pipeline to an active and available pipeline server.
+* You have previously triggered a pipeline run.
+
+.Procedure
+. From the {productname-short} dashboard, click *Data Science Pipelines* -> *Pipelines*.
++
+The *Pipelines* page opens.
+. From the *Project* list, select the project whose pipeline run step logs you want to view.
+. For the pipeline that you want to see logs for, click *Expand* (image:images/rhoai-expand-icon.png[]).
```

Comment:
Could change 'see' to 'view' to be consistent.

Reply:
Fixed.